### PR TITLE
Gov/experience page

### PR DESCRIPTION
**Pull request by GovernessS**
Will be adding more to this branch later - commits will all focus on this section of the site!

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -51,29 +51,77 @@
...
+  position: absolute;
+  right: 500px;
+  margin-top: 200px;
+}
+
+/* Create three equal columns that floats next to each other */
+.column {
+  float: left;
+  margin-left: 30px;
+  width: 300px;
```
Comment (sbahaud) [inline portfolio/src/main/webapp/style.css:90]: alphabetize the properties

File: portfolio/src/main/webapp/about.html [hunk 2]
```diff
@@ -9,6 +9,11 @@
 
 <body>
 
+    <!-- Notes for page: I am a rising junior at the University of Minnesota, 
+    majoring in Industrial and Systems Engineering with a minor in 
+    Computer Science! This summer, I will be a Program Manager intern 
+     Microsoft. I aspire to become a Product Manager (Technical or Associate).-->
+
```
[the remark this answers is not in the document]
Comment (GovernessS) [reply portfolio/src/main/webapp/about.html:16]: Done

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -37,36 +38,22 @@ <h1>Governess Simpson</h1>
...
-        Resume:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-
-      <div id="greeting-container"></div>
-
-    </div>
+    <div id="linkedin">
+        <a href="https://www.linkedin.com/in/governess-simpson/">
+        <img border="0" alt="https://www.linkedin.com/"
+         src="/images/linkedin-icon.png" width="100" height="100"></a></div>
```
[the remark this answers is not in the document]
Comment (GovernessS) [reply portfolio/src/main/webapp/index.html:52]: Done

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (GovernessS) [reply portfolio/src/main/webapp/style.css:90]: Done

File: portfolio/src/main/webapp/experience.html [hunk 4]
```diff
@@ -24,11 +24,76 @@
...
 
   <body>
-      <div id="content">
-      <p class="intro">
-        I am a rising junior at the University of Minnesota, majoring in Industrial and Systems Engineering with a minor in 
-         Computer Science! This summer, I will be a Program Manager intern at Microsoft. I aspire to become a Product Manager
-         (Technical or Associate).</p>
+          <div class="row">
+            <div class="column side">
+                <div id="code2040">
```
[the remark this answers is not in the document]
Comment (GovernessS) [reply portfolio/src/main/webapp/experience.html:29]: Done

File: portfolio/src/main/webapp/experience.html [hunk 5]
```diff
@@ -24,11 +24,76 @@
...
+                    <img class="top_row" class="sizing" src="/images/code2040-icon.jpg"  height="200" width="200">
+                </div>
```
[the remark this answers is not in the document]
Comment (GovernessS) [reply portfolio/src/main/webapp/experience.html:31]: Done

**Pull request merged**